src/components/KMultiselect/KMultiselect.vue

```diff
@@ -1002,6 +1003,7 @@ onMounted(() => {
     .k-multiselect-input {
       display: inline-block;
       position: relative;
+      width: 100%;
```

Comment:
I'm unsure if this fixes in all scenarios. When I inspect the multiselect item template slot example, I see this rule taking precedence
![image](https://user-images.githubusercontent.com/2229946/223145043-6329e8c4-004c-4c79-9f8d-10088eb4a056.png)

Reply:
The added `width` property is actually targeting the wrapper element - which is the one limiting input elements width. I tested the scenario when `item-template` slot is passed - it works as expected.